src/components/Chat/InputMessage.js

```diff
@@ -83,7 +83,7 @@ const InputMessage = () => {
             disabled={isDisableButton()}
             onPress={handleSendMessage}>
             <IconSend
-              fillBackground={isDisableButton() ? COLORS.gray9 : COLORS.holyTosca}
+              fillBackground={isDisableButton() ? colors.gray1 : colors.bondi_blue}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color values have been changed. Ensure that the new colors (`colors.gray1` and `colors.bondi_blue`) are equivalent to the old ones (`COLORS.gray9` and `COLORS.holyTosca`) in terms of user experience and accessibility.
```diff
- fillBackground={isDisableButton() ? COLORS.gray9 : COLORS.holyTosca}
+ fillBackground={isDisableButton() ? colors.gray1 : colors.bondi_blue}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->